marathon.json

```diff
@@ -0,0 +1,13 @@
+{
+    "id": "hdfs",
+    "container": {
```

Comment:
Why fill in the Docker container info without an image? If you're just launching the non-dockerized scheduler, just specify the URI(s) and leave containerInfo alone.